spec/models/night_finder_spec.rb

```diff
@@ -0,0 +1,27 @@
+require "rails_helper"
+
+RSpec.describe NightFinder do
+  before do
+    couch = create(:couch)
+    couch.nights << create(:night, date: Date.current)
+    couch.nights << create(:night, date: Date.tomorrow)
+    couch.nights << create(:night, date: Date.tomorrow.tomorrow)
+    @nights_params = { "check_in" => Date.current.to_s, "check_out" => Date.tomorrow.tomorrow.to_s, "couch_id" => couch.id }
+  end
+
+  describe ".find" do
+    it "returns the nights in a given range for a couch" do
```

Comment:
@tmikeschu for this test - maybe we do one check to test that the nights which are returned are for the couch we expect and within the date range we expect.   I'm not suggesting anything exhaustive, because that should be tested by the scopes in the `Night` class.  But it seems that performing one check for couch and date would make sure the NightFinder is calling the Night class' scopes properly.  Thoughts?